posts/2020-03-04-Rust-survey-2019.md

```diff
@@ -0,0 +1,167 @@
+
+We also asked users how they would rate their Rust Expertise - there is a clear peak around "7".
+
+![How would you rate your Rust expertise](/images/2020-03-RustSurvey/13-Rust-Expertise-and-Time-In-Rust.png)
+
+We compared these statistics with time spent using Rust. Rust expertise is highest at and after 1-2 years of experience.
+
+![How difficult are Rust concepts](/images/2020-03-RustSurvey/15-Rust-Concepts-Difficulty.png)
+
+When it comes to Rust concepts, the results were similar to last year. Enums, Cargo, and Collections were relatively easy concepts, followed by Pattern Matching, Iterators, Modules, Closures, and Traits, and Concurrency. The most challenging concepts were Ownership, Generics & Trait Bounds, FFI, Async I/O (new this year), Unsafe, Macros, and Lifetimes.
+
+## **Rust toolchain**
+
+![Which Rust version do you use](/images/2020-03-RustSurvey/16-Which-Rust-Version.png)
+
+The vast majority of users use the current stable version of Rust (63%). Surprisingly, the number of users using the Nightly compiler in their workflow is down at 20%. Last year it was at over 56%.
+
+![Preferred install method](/images/2020-03-RustSurvey/17-Preferred-Install-Method.png)
+
+We still see a very large percentage of users using `rustup` to install Rust, followed by Linux distros.
+
+This year we asked about the upgrade to Rust 2018. 92.8% of our users who responded to the survey are using the 2018 edition of Rust. 85% of users said it took little work do do so.
+
+![Comparative improvement](/images/2020-03-RustSurvey/18-Comparative-Improvement.png)
+
+Users indicated that async I/O improved the most in Rust 2018, no doubt related to the release of [async-await](https://blog.rust-lang.org/2019/11/07/Async-await-stable.html), while GUI development and learning curve improved the least. Clearly we still have much to do to improve Rust's learning curve.
+
+## **Rust workflow**
+
+==== Add Chart Here ====
+
+![Which platform are you developing on]()
+
+51% of Rust users developed on Linux, while 24% developed on Windows, and 23% developed on macOS.
+
+Linux continues to be a powerhouse among Rust developers, holding on to roughly 80% of Rust developers.  Windows usage has grown slightly from 31% last year to 34% this year, its second year in a row of growth.
+
+==== Add Chart Here ====
+
+![Which platforms are you developing for]()
+
+Linux and Windows continue to dominate as targets for Rust applications - 55% of Rust users targeted Linux and 24.5% targeted Windows. Following close behind Windows are macOS and Web Assembly at 22% each.
+
+==== Add Chart Here ====
+
+![What editors do you use]()
+
+Vim and VSCode continue to be the most popular editors (23.6% and 34.9% respectively) by far.
+
+## Feeling welcome
+
+We received a lot of feedback on how we can improve Rust and making it feel more welcome to more people. We can't include all of it here, so here is a summary of some of the feedback that stood out to us.
```

Comment:
I didn't keep a set of examples, so I've went through all of the feedback again and compiled three lists of quotes in a document that we can pull from. A list of short one line feedback and more constructive feedback. I also included all the positive feedback, I don't know if we should include that since it might seem self serving, but it's nice to read and helps put the other feedback in a broader perspective.

https://paper.dropbox.com/doc/What-actions-could-we-take-to-make-you-feel-more-welcome--AxMjBSNGm9pz9QUdb5CIIdTiAQ-vsqRYS9QYiOw0fKh1I2qK